test/e2e/framework/test_context.go

```diff
@@ -128,4 +129,5 @@ func RegisterFlags() {
 	flag.BoolVar(&TestContext.GatherMetricsAfterTest, "gather-metrics-at-teardown", false, "If set to true framwork will gather metrics from all components after each test.")
 	flag.StringVar(&TestContext.OutputPrintType, "output-print-type", "hr", "Comma separated list: 'hr' for human readable summaries 'json' for JSON ones.")
 	flag.BoolVar(&TestContext.DumpLogsOnFailure, "dump-logs-on-failure", true, "If set to true test will dump data about the namespace in which test was running.")
+	flag.IntVar(&TestContext.Scale, "scale", 1, "(warning: for use by experts only!) tests that support this option will multiply their fury!")
```

Comment:
What I mean is, "scale" can mean so many things in kubernetes, and if I just saw this file I would have no idea what this parameter actually does :P